platform/source/lib/ax/platform/minion_management/spot_instance_option_manager.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```

Comment:
NIT: you can move it under ax.platform.minion_manager. minion_manager/minion_management is a bit confusing. Also, you don't even need to expose this class outside the package as it is only used by minion manager